data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,63 @@ mission "Remnant: Scent of Rain 1"
+			`	"Our scouts are reporting a stranded Korath worldship in Vaticanus, a Kas'lor Ik 582 to be precise. If it were a typical warship obstructing our operations we would request you track it down and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board?" his signs shift from a questioning tone to a confident one. "After you are done we can disarm the ship and fix their jump drive. If they adhere to their typical doctrine they should decide to head back home."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have no intentions of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Together we can ensure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment and his gestures take on a staccato phrasing. "I feared my ask was too much. I understand the sentiment. Perhaps I was naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Vaticanus"
+		ship "Kas'lor Ik 582 (Stranded)" "Fligrookor"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on enter "Vaticanus"
+		dialog
+			`As you enter the system your scanners pick the target ship, and reveal an impressive amount of weaponry and military hardware. The environmental data reveals a small town's worth of people, many whose size indicates they are probably children.`
+	on abort
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		conversation
+			branch failed
+				not "Remnant: Will Not Someone Please Think Of The Children: aborted"
+			`The remnants are puzzled at your aborting of this mission, but they tell you they will send someone else to do this work. They warn you they will not propose any more of the sort to you again.`
+				decline
+			label failed
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
+	on complete
+		"remnant: disable and save count" ++
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matter. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he sings with a content smile on his face.`
```

Comment:
Capital after the speech ends with a full stop?
Also, did you definitely mean `sings` or was it supposed to be `signs`?